mod/LogicRuleMetadata.cs

```diff
@@ -0,0 +1,29 @@
+﻿using ArchipelagoRandomizer.InGameTracker;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ArchipelagoRandomizer;
+
+public class LogicRuleMetadata
+{
+    public class LogicMetadata
+    {
+        public string trackerCategoryName; // shown directly to the user in the ship log / in-game tracker
+        public string slotDataOption; // the one you added to Options.py and __init__.py
+        public string logicCategory;  // the one you used in items.jsonc, locations.jsonc and connections.jsonc
+    };
+
+    private static LogicMetadata FeldsparQuickAccess = new LogicMetadata {
+        trackerCategoryName = "Feldspar via Quick Access",
```

Comment:
looks like `trackerCategoryName` is unused? (which is probably correct; I doubt we'll be adding a whole top-level section to AP Checklist mode for any of these options)

Reply:
I think this is vestigial from when I was trying to go more down the route of how the `logsanity` options are added. This piece can definitely go